detect_secrets/plugins/keyword.py

```diff
@@ -49,30 +49,31 @@
 BLACKLIST_REGEX = re.compile(
     # Followed by double-quotes and a semi-colon
     # e.g. private_key "something";
+    # e.g. private_key 'something';
     r'|'.join(
-        '(.*?){}(.*?)"(.*?)";'.format(
+        '{}(.*?)("|\')(\\S*?)(\'|");'.format(
             value,
         )
         for value in BLACKLIST
     ) + '|' +
     # Followed by a :
     # e.g. token:
     r'|'.join(
-        '(.*?){}[:](.*?)'.format(
+        '{}:'.format(
             value,
         )
         for value in BLACKLIST
     ) + '|' +
     # Follwed by an = sign
     # e.g. my_password =
     r'|'.join(
-        '(.*?){}(.*?)=(.*?)'.format(
+        '{}(.*?)='.format(
```

Comment:
Same for this: we need to capture the secret here, instead of the content in between the keyword and the equal sign.